cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -857,7 +857,7 @@ define([
 
         beforeEach(function() {
             TemplateHelpers.installTemplate('content-group-details', true);
-            this.model = new GroupModel({name: 'Content Group', id: 0});
+            this.model = new GroupModel({name: 'Content Group', id: 0, courseOutlineUrl: "CourseOutlineUrl"});
```

Comment:
Note to reviewers: This is needed because now a URL is passed to the template instead of a complete HTML blob (which before could be missing without causing tests to fail).